stream-android-core/src/main/java/io/getstream/android/core/api/model/value/StreamHttpUrl.kt

```diff
@@ -15,26 +15,26 @@
  */
 package io.getstream.android.core.api.model.value
 
-import io.getstream.android.core.annotations.StreamCoreApi
+import io.getstream.android.core.annotations.StreamPublishedApi
 import java.net.URI
 
 /**
  * Represents an HTTP URL.
  *
  * @property rawValue The raw value of the HTTP URL.
  */
-@StreamCoreApi
+@StreamPublishedApi
 @JvmInline
-value class StreamHttpUrl(val rawValue: String) {
-    companion object {
+public value class StreamHttpUrl(public val rawValue: String) {
+    public companion object {
```

Comment:
Similar question as above, although this is not yet used in feeds. Tho, I imagine this **is** supposed to be passed from clients, right? Same for `StreamWsUrl`.